pkg/providers/aws/rds/rds.go

```diff
@@ -57,6 +57,7 @@ type Instance struct {
 	PublicAccess              defsecTypes.BoolValue
 	Engine                    defsecTypes.StringValue
 	IAMAuthEnabled            defsecTypes.BoolValue
+	DeletionProtection        defsecTypes.BoolValue
```

Comment:
Adding the `DeletionProtection` property to the RDS Instance struct so we can check it with rego.